scipy/stats/_stats_py.py

```diff
@@ -440,6 +440,12 @@ def mode(a, axis=0, nan_policy='propagate'):
     The mode of object arrays is calculated using `collections.Counter`, which
     treats NaNs with different binary representations as distinct.
 
+    .. deprecated:: 1.9.0
+        Support for object arrays has been deprecated and will be removed in
+        the second release after SciPy 1.9.0.
+        `pandas.DataFrame.mode <https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.mode.html>`_
```

Comment:
`scipy/stats/_stats_py.py:446:80: E501 line too long (106 > 79 characters)`

Reply:
Done.

Reply:
Well, 
![image](https://user-images.githubusercontent.com/6570539/154614366-b43e2b1a-b6a8-45a8-aca9-ece0be3d7fd9.png)

Not sure how to resolve it other than removing the `noqa` and ignoring the complaint. Doesn't `lint_diff.py` only ignore  new PEP8 errors introduced by the PR (in theory)? So this wouldn't become a PEP8 failure in main, right?

Reply:
Oops, I didn't see this. Should be fixed now.

Reply:
Works for me: https://43377-1460385-gh.circle-artifacts.com/0/html-scipyorg/reference/generated/scipy.stats.mode.html#scipy.stats.mode

Reply:
We can just ignore lint errors in the whole documentation block (otherwise the `noqa` will keep appearing in the link). I don't think it should be a problem.

Reply:
Finally working!